spec/controllers/book_suggetions_controller_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe BookSuggestionsController, type: :controller do
+  describe 'POST #create' do
+    context 'When creating a valid book_suggestion' do
+      let!(:book_suggestion) { create(:book_suggestion) }
+
+      it 'creates a new book suggestion' do
+        expect do
+          post :create, params: { book_suggestion: book_suggestion.attributes }
+        end.to change { BookSuggestion.count }.by(1)
+      end
+
+      it 'responds with 201 status' do
+        post :create, params: { book_suggestion: book_suggestion.attributes }
+        expect(response).to have_http_status(:created)
+      end
+    end
+
+    context 'When creating an invalid book_suggestion' do
+      let!(:book_suggestion) do
+        bs = FactoryBot.build(:book_suggestion, author: '')
```

Comment:
All FactoryBot methods are already included, you can call it without `FactoryBot` => `build(:book_suggestion, author: '')`